index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header id="header" >
+      <img id="header-img" src="Logo.png" alt="Logo">
+      <nav id="nav-bar"><a class="nav-link" href="#conocenos">Button 1</a> <a class="nav-link" href="#video">Button 2</a><a class="nav-link" href="#precios">Button 3</a></nav>
+    </header>
```

Comment:
Hi, @HFG43!! 👋
- Well done on making inline comments with suggestions and reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better to use `nav` nested of your `header`". 

- [ ] I can see some grammatical errors, and also you haven't closed the bracket `(`. Kindly use [Grammarly](https://app.grammarly.com/) so that there are no spelling mistakes and other grammatical mistakes. It is more professional to check the spelling while giving any suggestion in every aspect of our job and profession.